ui/js/component/app/view.jsx

```diff
@@ -4,8 +4,14 @@ import Header from "component/header";
 import Theme from "component/theme";
 import ModalRouter from "modal/modalRouter";
 import lbry from "lbry";
+import _ from "lodash";
```

Comment:
This is easy, but it is not simple.

(This is a distinction you will see me make often. Please watch "Simple Made Easy" by Rich Hickey if you have never seen it. It's relatively short and funny and covers one of the most important lessons in programming. You too @liamcardenas)

If all we need is a throttle method, rather than bringing in an entire library, I'd propose simply bringing that method or an analogous one in directly and making it a core part of our code.

From a quick Google, there's no shortage of implementations for throttling javascript events. Here are several in one thread: https://stackoverflow.com/questions/27078285/simple-throttle-in-js